definitions/consented/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -4880,7 +4880,7 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y",
-    "PageShowCondition": "currentUserCaseRole=\"[APPSOLICITOR]\""
+    "PageShowCondition": "currentUserForConsentedCaseRoleLabel=\"APPSOLICITOR\""
```

Comment:
Whats reason using currentUserForConsentedCaseRoleLabel instead of user case role? and currentUserForConsentedCaseRoleLabel is text field

Reply:
UserCaseRole has been used further down  - so do not want to manipulate that. Recent XUI change - we can't use special characters in PageShow os FieldShow condition.